tests/python_tests/samples/conftest.py

```diff
@@ -171,21 +171,20 @@ def convert_model(request):
     model_cache = os.path.join(models_cache, model_id)
     model_path = os.path.join(model_cache, model_name)
     model_args = MODELS[model_id]["convert_args"]
-    model_hf_cache = os.environ.get("HF_HOME", os.path.join(model_cache, "hf_cache"))
     logger.info(f"Preparing model: {model_name}")
     # Convert the model if not already converted
     if not os.path.exists(model_path):
         logger.info(f"Converting model: {model_name}")
+        sub_env=os.environ.copy()
         command = [
             "optimum-cli", "export", "openvino",
             "--model", model_name, 
-            "--cache_dir", model_hf_cache, 
```

Comment:
Why is it removed, should we still try to save time by using it?
I guess this was the only usage of `HF_HOME` so it should be removed from worflow

Reply:
that change doesn't affect on HF cache usage, optimum-cli will use the environment variables directly (string 187)